ui/app/routes/vault/cluster/auth.js

```diff
@@ -25,19 +25,31 @@ export default class AuthRoute extends ClusterRouteBase {
   }
 
   async model(params) {
+    const clusterModel = this.modelFor('vault.cluster');
     const wrapped_token = params?.wrapped_token;
     if (wrapped_token) {
-      const clusterModel = this.modelFor('vault.cluster');
-      await this.unwrapToken(wrapped_token, clusterModel.id);
+      // log user in directly (i.e. no form interaction) via URL query param
+      const authResponse = await this.unwrapToken(wrapped_token, clusterModel.id);
+      return { clusterModel, unwrapResponse: authResponse };
     }
-    return super.model(...arguments);
```

Comment:
`super.model()` returned the cluster model previously 
https://github.com/hashicorp/vault/blob/c1754f5f97309b0a5714036c54e66ace7934785d/ui/app/routes/vault/cluster/cluster-route-base.js#L16-L18